src/routes/api/playlists/user/+server.ts

```diff
@@ -5,7 +5,8 @@ import type { RequestHandler } from './$types';
 export const GET = (async (event) => {
     const { uid } = await authenticate(event);
     const playlists = await readPath(`/playlists/user/${uid}`) || {};
-    return json(Object.values(playlists), { headers: [ ['Access-Control-Allow-Origin', "*"] ]});
+    const modifiedPlaylists = Object.values(playlists).map(playlist => ({ ...playlist, words: playlist.words.map(word => Object.values(word).map(letters => (letters === false ? null : letters)))}));
```

Comment:
This can also be simplified slightly (also the `letters` -> `letter` thing is just a nitpick)

```suggestion
    const modifiedPlaylists = Object.values(playlists).map(playlist => ({ ...playlist, words: playlist.words.map(word => Object.values(word).map(letter => (letter || null)))}));
```